src/reducers/edititem/index.js

```diff
@@ -0,0 +1,26 @@
+import {
+  EDIT_ITEM_CREATE,
+  EDIT_ITEM_UPDATE,
+  EDIT_ITEM_REMOVE
+} from 'actions/actiontypes'
+
+const initialState = {
+}
+
+const editItemReducer = (state = initialState, action) => {
```

Comment:
Could u explain what is happening here? aren't they all supposed to return a state? or am I mis understanding something. I'm not that familiar with redux haha

Reply:
nevermind i get it haha

Reply:
mark this resolved.